pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/etterlatte/maler/barnepensjon/informasjon/BarnepensjonInformasjonDoedsfall.kt

```diff
@@ -77,31 +76,31 @@ object BarnepensjonInformasjonDoedsfall : EtterlatteTemplate<BarnepensjonInforma
                     )
                 }
 
+                title2 {
+                    text(
+                        Bokmal to "Hvordan søker du?",
+                        Nynorsk to "Korleis søkjer du?",
+                        English to "How do you apply?",
+                    )
+                }
                 showIf(borIutland.not().and(erOver18aar.not())) {
-                    title2 {
+                    paragraph {
                         text(
-                            Bokmal to "Hvordan søker du?",
-                            Nynorsk to "Korleis søkjer du?",
-                            English to "How do you apply?",
+                            Bokmal to "Du finner informasjon og søknad på $BARNEPENSJON_URL.",
+                            Nynorsk to "Du finn informasjon og søknad på $BARNEPENSJON_URL.",
+                            English to "You will find information and the application form at $BARNEPENSJON_URL.",
```

Comment:
Skulle man brukt engelsk url her?

Reply:
Gjelder for de andre tilfellene der man bruker `BARNEPENSJON_URL`også

Reply:
https://github.com/navikt/pensjonsbrev/pull/698/commits/726285a4b0903106a4ab58213d24e708c8db1110